experimenter/experimenter/jetstream/client.py

```diff
@@ -365,3 +369,21 @@ def get_population_sizing_data():
     sizing_data = get_sizing_data(suffix="latest")
     sizing = SampleSizes.parse_obj(sizing_data) if sizing_data is not None else {}
     return {"v1": sizing}
+
+
+def get_cached_population_sizing_data() -> SampleSizes | None:
+    try:
+        sizing_data = cache.get(settings.SIZING_DATA_KEY)
+        if not isinstance(sizing_data, str):
+            raise TypeError(f"expected str, got {type(sizing_data)}")
+
+        sizing_data = SampleSizes.parse_raw(sizing_data)
+    except Exception as e:
+        logger.warning(
```

Comment:
Seems like an error since it indicates bad sizing data, no?
```suggestion
        logger.error(
```